var/spack/repos/builtin/packages/sst-core/package.py

```diff
@@ -7,38 +7,72 @@
 
 
 class SstCore(AutotoolsPackage):
-    """The Structural Simulation Toolkit (SST) was developed to explore
-    innovations in highly concurrent systems where the ISA, microarchitecture,
-    and memory interact with the programming model and communications system"""
+    """The Structural Simulation Toolkit (SST) core
+       provides a parallel discrete event simulation (PDES)
+       framework for performing architecture simulations
+       of existing and proposed HPC systems"""
 
-    homepage = "http://sst-simulator.org/"
-    url      = "https://github.com/sstsimulator/sst-core/releases/download/v8.0.0_Final/sstcore-8.0.0.tar.gz"
-    git      = "https://github.com/sstsimulator/sst-core.git"
+    homepage = "https://github.com/sstsimulator"
+    git = "https://github.com/sstsimulator/sst-core.git"
+    url = "https://github.com/sstsimulator/sst-core/releases/download/v9.1.0_Final/sstcore-9.1.0.tar.gz"
 
-    version('develop', branch='devel')
-    version('8.0.0', sha256='34a62425c3209cf80b6bca99cb0dcc328b67fb84ed92d5e6d6c975ad9319ba8a')
+    version('9.1.0', sha256="cfeda39bb2ce9f32032480427517df62e852c0b3713797255e3b838075f3614d")
+    version('devel',   branch='devel')
```

Comment:
```suggestion
    version('develop',   branch='devel')
```
This allows Spack to sort the version correctly